aloha-core/src/main/scala/com/eharmony/aloha/dataset/csv/CsvRowCreator.scala

```diff
@@ -22,19 +22,19 @@ import scala.util.{Failure, Success, Try}
  * @tparam A the input type that is transformed into CSV output.
  */
 final case class CsvRowCreator[-A](features: FeatureExtractorFunction[A, String], headers: Vector[String], separator: String = ",")
-    extends RowCreator[A] {
-    def apply(data: A) = {
+    extends CharSeqRowCreator[A] {
+    def apply(data: A): (MissingAndErroneousFeatureInfo, String) = {
```

Comment:
Is there a reason you changed from `CharSequence` to `String`?

Reply:
Nope.  Just a string because it's a more specific type and `B` is covariant, so why not give the more specific type.

Reply:
Then why not get rid of `CharSequence` everywhere?  Frankly I never understood why we had it in the first place.